modules/ducktests/src/main/java/org/apache/ignite/internal/ducktest/tests/rebalance/DataGenerationApplication.java

```diff
@@ -57,16 +58,16 @@
      * @param entryCnt Entry count.
      * @param entrySize Entry size.
      */
-    private void generateCacheData(String cacheName, int entryCnt, int entrySize) {
+    private void generateCacheData(String cacheName, int entryCnt, int entrySize, int startKey) {
         int logStreamedEntriesQuant = (int)Math.pow(10, (int)Math.log10(entryCnt) - 1);
 
+        int streamed = 0;
```

Comment:
As far as I can understand streamed always equal to i+1.
Can we just get rid of this variable?